intersection2Arrays2PointerApproach.java

```diff
@@ -0,0 +1,38 @@
+//TC for sorted = O(mlogm)+O(nlogn)
+//SC =O(1)
```

Comment:
SC is O(log k ) to O(k) where k is max(m,n)  
here space used by result should also be considered